src/modules/create-wallet/components/CreateWalletMnemonicPhrase.vue

```diff
@@ -90,9 +90,9 @@
               @click.native="updateStep(2)"
             />
           </div>
-        </v-sheet>
+        </mew6-white-sheet>
         <mew-warning-sheet
-          class="mt-4 mb-0"
+          class="mt-5 mb-0"
```

Comment:
if you are changing this, can you make sure the warning mt-5 is the same on other create wallet methods?